lib/bugsnag/report.rb

```diff
@@ -23,13 +23,16 @@ class Report
     attr_accessor :api_key
     attr_accessor :app_type
     attr_accessor :app_version
+    attr_accessor :breadcrumbs
     attr_accessor :configuration
     attr_accessor :context
     attr_accessor :delivery_method
     attr_accessor :exceptions
     attr_accessor :hostname
     attr_accessor :grouping_hash
+    attr_accessor :message
     attr_accessor :meta_data
+    attr_accessor :name
```

Comment:
After digging through the existing code some more, it looks like it would be easier to define a plain `name` method (instead of `attr_accessor :name`) that returns `exceptions[0][:errorClass]` as that would be consistent (because the `exceptions` attribute also makes the exception go through the `#error_class` method to find its error class or "name", which your code further below doesn't do now).

Reply:
Keeping the `message` and `name` as it is now allows the user to override how the follow-up breadcrumb will be displayed.  If they want to assign a custom name/message that will allow them to navigate breadcrumbs more easily they should be able to.